doc/doxygen/snippets/CMakeLists.txt

```diff
@@ -6,9 +6,11 @@ set(sources
   lambda_mesh_data_set.cc
   ref_el.cc
   vtk_writer.cc
+  mesh_utils.cc
 )
 
 add_library(docsnippets ${sources})
 get_target_property(OUT lf.base INCLUDE_DIRECTORIES)
 target_link_libraries(docsnippets PUBLIC Eigen3::Eigen lf.base)
-target_compile_features(docsnippets PUBLIC cxx_std_17)
\ No newline at end of file
+target_link_libraries(docsnippets PUBLIC Eigen3::Eigen lf.base lf.mesh)
```

Comment:
you're creating a second target called docsnippets, I think line 14 can be deleted?

Reply:
Line 14 is deleted.